core/templates/dev/head/components/common-layout-directives/navigation-bars/top-navigation-bar.directive.html

```diff
@@ -295,4 +295,8 @@
     letter-spacing: .21px;
     line-height: 38px;
   }
+  .oppia-navbar-profile > li:hover {
+     background-color: #ffffff;
```

Comment:
Use 3 character hexadecimal notation wherever possible like in this case, it can be `#fff`.

Reply:
Done